Alchemy/CodeChain/DefPrimitives.h

```diff
@@ -186,6 +186,13 @@ static PRIMITIVEPROCDEF g_DefPrimitives[] =
 			"(for var from to exp) -> value of last expression",
 			NULL,	0,	},
 
+		{	"gammaScale",		fnMathNumerals,	FN_MATH_GAMMA_SCALE_NUMERALS,
+			"(gammaScale var inMin inMax outMin outMax [gamma=1.0]) -> int or double of gammascaled value relative to input and output range.\n"
+			"Returns int if both outMin and outMax are ints, otherwise returns a double. Gamma specifies an exponent relationship curve between input and output ranges.\n"
```

Comment:
Most multi line help messages have a blank line between the signature and the extended text which helps with readability in the console.

```suggestion
			"(gammaScale var inMin inMax outMin outMax [gamma=1.0]) -> scaled value (int or real)\n\n"
			
			"Scales var from range inMin to inMax to the output scale outMin to outMax.\n"
			"Returns int if both outMin and outMax are ints, otherwise returns a real. Gamma specifies an exponent relationship curve between input and output ranges.\n"
```